src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderService.java

```diff
@@ -107,9 +109,9 @@ public ProbationRecord getProbationRecord(String crn, boolean applyDocumentFilte
                 probationRecord.setAssessment(findMostRecentByStatus(assessments).orElse(null));
             });
         } catch (OffenderNotFoundException e) {
-            log.info("assessment data missing from probation record (CRN '{}' not found in oasys)", crn);
```

Comment:
Removed these log statements as suggested. We'll still have the error level logging in the AssessmentRestClient